launcher/launcher_test.go

```diff
@@ -748,7 +775,7 @@ func TestHttpProtoDefaultsToCorrectHostAndPort(t *testing.T) {
 
 	shutdown, _ := ConfigureOpenTelemetry(
 		WithLogger(logger),
-		WithExporterEndpoint(strings.TrimPrefix(ts.URL, "http://")),
+		WithExporterEndpoint(ts.URL),
```

Comment:
Instead of trimming the http prefix in the test, we're now trimming in the code itself